airtime/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if (IOS)
 endif()
 
 # Disable warnings introduced in Xcode 7.3
-add_compile_options(-Wno-logical-not-parentheses)
+add_compile_options(-Wno-logical-not-parentheses -Wno-implicit-fallthrough)
```

Comment:
why add ths?

Reply:
The BoringSSL code has an issue that triggers this warning. It's probably fixed upstream, but I don't want to incur the time (and risk) of bringing in a new BoringSSL version as part of this project.